src/plugins/definition/schema.js

```diff
@@ -60,33 +60,19 @@ function normalizeDefinition(change, error) {
             break
         }
 
+        /* istanbul ignore next */
         console.log('Unhandled definition child_min_invalid:', child)
         break
 
-    case 'child_unknown':
-        // An invalid node following the See Also should be folded into the
-        // See Also.
-        if (index > 0) {
-            const nodeBefore = node.nodes.get(index - 1)
-            if (nodeBefore && nodeBefore.type === 'definition_seealso') {
-                change.moveNodeByKey(child.key, nodeBefore.key, nodeBefore.nodes.size)
-                break
-            }
-        }
-
-        // Unwrap any other nodes
-        change.unwrapBlockByKey(child.key)
-        break
-
     /* istanbul ignore next */
     default:
         console.warn('Unhandled definition violation:', code)
         break
     }
 }
 
+/* istanbul ignore next */
 function normalizeTerm(change, error) {
```

Comment:
Don't ignore functions, it will be very easy to forget about it when we add some normalizations.